indico/modules/rb_new/client/js/modules/calendar/actions.js

```diff
@@ -44,34 +52,45 @@ export function setMode(mode) {
     return {type: SET_MODE, mode};
 }
 
+async function fetchCalendarRooms(dispatch, state) {
+    const roomFilters = getRoomFilters(state);
+
+    if (!_.isEmpty(roomFilters)) {
```

Comment:
do you really want to do *nothing* when there are no filters?

Reply:
I think you need to at least return an empty list in that case since later you check the `.length` of the return value..